src/Build.UnitTests/BackEnd/MSBuild_Tests.cs

```diff
@@ -374,10 +374,11 @@ public void PropertyOverridesContainSemicolon()
             ObjectModelHelpers.CreateFileInTempProjectDirectory(
                 @"bug'533'369\Sub;Dir\ConsoleApplication1\ConsoleApplication1.csproj", @"
 
-                <Project DefaultTargets=`Build` ToolsVersion=`msbuilddefaulttoolsversion` xmlns=`msbuildnamespace`>
+                <Project DefaultTargets=`Build` xmlns=`msbuildnamespace`>
                   <PropertyGroup>
                     <Configuration Condition=` '$(Configuration)' == '' `>Debug</Configuration>
                     <Platform Condition=` '$(Platform)' == '' `>AnyCPU</Platform>
+                    <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
```

Comment:
Can you define a constant named something like `StandardTestTargetFrameworkVersion` so that we can update these in a single pass if we need to in the future?